src/Softeq.NetKit.Chat.Data.Persistent.Sql/Repositories/MemberRepository.cs

```diff
@@ -100,9 +144,24 @@ public async Task<Member> GetMemberByIdAsync(Guid memberId)
         {
             using (var connection = _sqlConnectionFactory.CreateConnection())
             {
-                var sqlQuery = @"SELECT *
-                                 FROM Members
-                                 WHERE Id = @memberId";
+                var sqlQuery = $@"
+                    SELECT 
+                        {nameof(Member.Id)},
+                        {nameof(Member.Email)},
+                        {nameof(Member.IsAfk)},
+                        {nameof(Member.IsBanned)},
+                        {nameof(Member.LastActivity)},
+                        {nameof(Member.LastNudged)},
+                        {nameof(Member.Name)},
+                        {nameof(Member.PhotoName)},
+                        {nameof(Member.Role)},
+                        {nameof(Member.SaasUserId)},
+                        {nameof(Member.Status)},
+                        {nameof(Member.IsActive)}
```

Comment:
Add {nameof(Member.IsDeleted)} after merging new version.

Reply:
We won't forget :)